golem-component-service-base/Cargo.toml

```diff
@@ -16,17 +16,22 @@ golem-common = { path = "../golem-common", version = "0.0.0" }
 golem-service-base = { path = "../golem-service-base" }
 golem-rib = { path = "../golem-rib" }
 golem-wasm-ast = { workspace = true }
+golem-worker-executor-base = { path = "../golem-worker-executor-base", version = "0.0.0" }
```

Comment:
This dependency is here for the blob storage. It feels very bad to have this dependency instead of some common crate though.

Reply:
That's a better place :)